dev-packages/application-package/src/application-package.ts

```diff
@@ -130,11 +130,11 @@ export class ApplicationPackage {
 
     async resolveExtensionPackage(extension: string): Promise<ExtensionPackage | undefined> {
```

Comment:
Add a comment to define `extension` as `the name of the extension's package as defined in "dependencies" (might be aliased)`.